cypress/integration/change-metadata.spec.ts

```diff
@@ -1,11 +1,18 @@
 describe("Changing metadata in the language info page", function () {
+  beforeEach(() => {
+    cy.clearLocalData();
+    cy.generateProject();
+  });
+
   it.skip("should find a button to press to add source by uploading file", function () {
-    const languageName = "Makah";
+    const languageName = "’Are’are";
     const authorName = "Eddie";
     const copyright = "© 2018 My Cool Organization";
 
     cy.visit("/customize");
 
+    // Wait for the languages to load
+    cy.wait(1000);
```

Comment:
Perhaps mock the keyboard response for this test?

Reply:
That could work! I'll need to find a way to implant languages into the Svelte application though. If making the IndexedDB generation and deletion blocking doesn't work, I'll investigate if mocking could be another approach to this problem.

Reply:
I think the keyboard response here is mocked out (the intercept gets set up in `cy.generateProject()`). My guess as to why this `cy.wait()` needs to now be here is that the mocked response doesn't get cached, and therefore Cypress needs time to intercept the request that the Svelte app sends out.